l10n_ro_hr/hr_contract_view.xml

```diff
@@ -62,30 +62,26 @@
 				</field>
```

Comment:
'False' fara '', in domeniu, si = in loc de ==...attrs="{'invisible': [('suspended', '=', False)]}", la fel si in perioada contract cu period_type in date_end...